### PR TITLE
[DOCS] Development community showcase page

### LINKED ISSUE #1313: [DOC] Page for community showcase
#### Describe the issue linked to the documentation
I would be nice to add a page to the About section on the website with community contributions (blog posts, reports, contributions, etc).

#### Suggest a potential alternative/fix
* Add community showcase page by adding a `sktime/docs/source/about/showcase.rst` file
* Remove Extra Resources section in `sktime/docs/source/user_guide/`
* Use https://github.com/sloria/sphinx-issues extension to link to GitHub users
* Adding a table for blogposts on that page would be a good start with 3 columns: title, date, short description, user. The first blog post to add could be:
  *  https://towardsdatascience.com/sktime-a-unified-python-library-for-time-series-machine-learning-3c103c139a55
  * https://medium.com/@jlenczuk/why-start-using-sktime-for-forecasting-8d6881c0a518

### PR DESCRIPTION
**Pull request by afzal442**
#### Reference Issues/PRs
Fixes #1313 

#### What does this implement/fix? Explain your changes.

This PR adds a community showcase page for blog posts
#### Does your contribution introduce a new dependency? If yes, which one?
No

##### For all contributions
- [ ] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [x] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

**Review by mloning — COMMENTED** [review]
Hi @afzal442, the page looks good to me, but there are a number of unrelated files in the PR (looks like you created the branch based on your previous work on the estimator table). Could you remove these files from the PR? Running `git rm <file>` should do what you need to do. 

We also need to add the showcase page to the index on the `docs/source/about.rst` page.

**Comment by RNKuhns** [comment]
@afzal442 it looks like the links might not be rendering in the table. I think the one linking the name and github link is missing the trailing underscore. 

Note sure what is up with the other one.

**Comment by afzal442** [comment]
something issues with sphinx-build error with md files.

qrcode_readthedocs org

**Comment by afzal442** [comment]
hi @RNKuhns, could you help me out why this got failed? Or any other suggestions if you have. https://sktime--1337.org.readthedocs.build/en/1337/about/showcase.html

File: .all-contributorsrc [hunk 1]
```diff
@@ -1162,15 +1162,6 @@
       "contributions": [
         "doc"
       ]
-    },
```
Comment (mloning) [inline .all-contributorsrc:1165]: Make sure you don't accidentally delete this change, run `checkout main -- .all-contributorsrc` and then `git commit` to undo the change

File: docs/source/about/showcase.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+   :header-rows: 1
+
+   * - Title
+     - Date
+     - Short Description
+     - User
+   * - `sktime: a unified python library for time series machine learning <https://towardsdatascience.com/sktime-a-unified-python-library-for-time-series-machine-learning-3c103c139a55>`_
+     -
+     -
+     - :user: `Alexandra Amidon <lynnssi>`_
```
Comment (mloning) [inline docs/source/about/showcase.rst:17]: we need to remove the space here
```suggestion
     - :user:`Alexandra Amidon <lynnssi>`_
```

File: docs/source/about/showcase.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+   * - `Why start using sktime for forecasting? <https://medium.com/@jlenczuk/why-start-using-sktime-for-forecasting-8d6881c0a518>`_
+     -
+     - Overview of sktime’s features that distinguish it from other forecasting frameworks
+     - `Joanna Lenczuk <https://github.com/joanlenczuk>`_
```
Comment (mloning) [inline docs/source/about/showcase.rst:21]:
```suggestion
     - :user:`Joanna Lenczuk <joanlenczuk>`_
```

File: docs/source/conf.py [hunk 4]
```diff
@@ -383,6 +383,10 @@ def adds(pth):
 
 
 # -- Extension configuration -------------------------------------------------
+# extensions = ["sphinx_issues"]
+
+# GitHub repo
+# issues_github_path = "sloria/marshmallow"
```
Comment (mloning) [inline docs/source/conf.py:389]: these changes are not needed
```suggestion
```

**Review by mloning — COMMENTED** [review]
Hi @afzal442, the failure was unrelated to your changes. I re-started the checks. 

Perhaps we should remove the date column, as the table seems a bit crowded? Or is there a way to fix the column widths of the individual columns? Alternatively we could use panels instead of a table, that would allow to add images too.

File: .all-contributorsrc (hunk 1 above)
Comment (afzal442) [reply .all-contributorsrc]: Sure, thanks.

**Comment by afzal442** [comment]
> Alternatively we could use panels instead of a table, that would allow to add images too.

Sounds good. will look into it.

File: docs/source/about/showcase.rst [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+.. list-table::
+   :widths: 25 50 25
+   :header-rows: 1
+
+   * - Title
+     - Short Description
+     - User
+   * - `sktime: a unified python library for time series machine learning <https://towardsdatascience.com/sktime-a-unified-python-library-for-time-series-machine-learning-3c103c139a55>`_
+     -
+     - :user:`Alexandra Amidon <lynnssi>`_
```
Comment (mloning) [inline docs/source/about/showcase.rst:15]: Hey @afzal442 - the users in the table don't render correctly: https://sktime--1337.org.readthedocs.build/en/1337/about/showcase.html
```suggestion
     - :user:`lynnssi`
```

**Review by mloning — COMMENTED** [review]
Hi @afzal442, let's get this in before our next release, I just have one minor comment

File: docs/source/about/showcase.rst (hunk 5 above)
Comment (mloning) [reply docs/source/about/showcase.rst:15]: this should also work
```
:user:`Alexandra Amidon <lynnssi>`
```

**Comment by afzal442** [comment]
@mloning https://sktime--1337.org.readthedocs.build/en/1337/about/showcase.html

**Review by mloning — APPROVED** [review]
Thanks @afzal442

**Comment by mloning** [comment]
Now merged @afzal442 🎉
